ui/lib/core/addon/components/confirm-action.js

```diff
@@ -82,6 +80,8 @@ export default class ConfirmActionComponent extends Component {
       confirmAction();
       // toggle
       this.showConfirm = !this.showConfirm;
```

Comment:
Re: comment below on test - I think the chevron isn't toggling correctly because `actions.close()` calls `this.toggleConfirm` and then we change the tracked variable again here in on line 84/82. 
So I think this line should be removed: `this.showConfirm = !this.showConfirm;`

Reply:
Yeah, that was a nest I think I created for myself. Keeping the old functionality and fixing it and basically creating a loop of untoggling toggling. Thank you for the catch.